### DISCUSSION TITLE
Filter action by event return old data

### DISCUSSION THREAD

**Discussion by mbiagetti** [community/community#24624] [Apps, API and Webhooks]
<p>when executing the API</p>
<pre><code class="lang-auto">curl -s "https://api.github.com/repos/${INPUT_OWNER}/${INPUT_REPO}/actions/runs?event=repository_dispatch" \
      -H "Accept: application/vnd.github.v3+json" \
      -H "Authorization: Bearer ${INPUT_TOKEN}" | jq '.workflow_runs[0]
</code></pre>
<p>The data returned is not updated (old data)</p>

**Comment by ronek22** [comment]
<p>I have the same issue</p>

**Comment by mbiagetti** [comment]
<p>Seems the filter is not working, so we fix removing the filter and adding the <code>per_page=1</code></p>

**Comment by mwilck** [comment]
<p>My issue <a href="https://github.community/t/recent-workflow-runs-are-mistakenly-hidden-when-filtering/244459" class="inline-onebox">Recent workflow runs are mistakenly hidden when filtering</a> is the same, I suppose?</p>

**Comment by gthb** [comment]
<p>I get this problem when filtering by branch, presumably the same problem.</p>
<p>When I try the same filtering in the web UI, I get a banner saying “We are having problems searching workflow runs. The results may not be complete.”</p>
<p>Yet nothing is indicated on <a href="http://status.github.com" rel="noopener nofollow ugc">status.github.com</a> :face_with_monocle:</p>

**Comment by mwilck** [comment]
<p>Issue still persists.</p>

**Comment by Mcb7778** [comment]
<p>Let’s just get everyone raspberries, lol</p>

**Comment by mwilck** [comment, marked as answer]
<p>Problem solved, AFAICS.</p>
